### PR TITLE
Js test summary

### LINKED ISSUE #473: Summarize test results when running all tests
It'd be nice to have a suite-by-suite summary of what was run, whether they passed, maybe how long they took, etc.

### PR DESCRIPTION
**Pull request by jsingle**
@thieman how's this look?
```
+------------+---------------------------------------+--------+---------------+
|   Suite    |              Description              | Result |    Time (s)   |
+------------+---------------------------------------+--------+---------------+
|  frontend  |      test of gcweb through django     |  PASS  | 10.8578848839 |
|    unit    |         basic nose unit tests         |  PASS  | 8.75790905952 |
| functional | Headless web-browser functional tests |  FAIL  | 145.212858915 |
|  jasmine   |           Jasmine (JS) tests          |  PASS  | 30.1717450619 |
+------------+---------------------------------------+--------+---------------+
TESTS FAILED
```

Resolves #473

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
     * Dusty now attempts to fix a known problem with boot2docker networking, resulting in up to a 10x improvement in network performance. See [this boot2docker issue](https://github.com/boot2docker/boot2docker/issues/1022) for more information.
 * **Misc**
     * Testing `once` commands that fail now cause the entire test run to fail immediately.
```
Comment (thieman) [inline docs/changelog.md]: Drive-by, can you rephrase this? It sounds like you are literally testing the once commands

**Comment by thieman** [comment]
LGTM, you should trim the time to 1 decimal point though

File: docs/changelog.md (hunk 1 above)
Comment (jsingle) [inline docs/changelog.md]: Sure

**Comment by jsingle** [comment]
Done

**Comment by thieman** [comment]
LGTM
